### PR TITLE
Change Hints and Notes to Admonitions

### LINKED ISSUE #122: Update All Hints and Notes in Text with Hint and Note Admonitions
This issue is set up to maintain the consistency of style across the lecture series.

### PR DESCRIPTION
**Pull request by HumphreyYang**
Hi @mmcky,

This resolves #122. Could you please merge it if it looks good to you?

Many thanks.

### CODE REVIEW

File: lectures/additive_functionals.md [hunk 1]
```diff
@@ -1194,14 +1194,15 @@ Then let's use the plots to  investigate how these densities evolve through time
...
+`scipy.stats.lognorm` expects you to pass the standard deviation
 first $(tH \cdot H)$ and then the exponent of the mean as a
 keyword argument `scale` (`scale=np.exp(-t * H2 / 2)`).
 
 * See the documentation [here](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.lognorm.html#scipy.stats.lognorm).
 
 This is peculiar, so make sure you are careful in working with the log normal distribution.
-
+```
 Here is some code that tackles these tasks
```
Comment (mmcky) [inline lectures/additive_functionals.md:1206]: @HumphreyYang super picky I know but I find it helpful to have a space after every directive -- make it a bit clearer to read the file.

**Review by mmcky — COMMENTED** [review]
thanks @HumphreyYang this is looking great. If you could just add a new line after the two directives that don't have them -- this is good to 🚀

File: lectures/additive_functionals.md (hunk 1 above)
Comment (HumphreyYang) [reply lectures/additive_functionals.md:1207]: @mmcky Thanks for the kind style advice. It does look better with directives separated out from the text. I will keep this in mind. Could you have a quick look at the latest commits in both repos to see if it is what you expected?

Many thanks.

**Review by mmcky — APPROVED** [review]
thanks @HumphreyYang

**Pull request merged**